src/bp-templates/bp-nouveau/includes/activity/ajax.php

```diff
@@ -91,6 +91,12 @@ function() {
 					'nopriv'   => true,
 				),
 			),
+			array(
+				'toggle_activity_notification_status' => array(
+					'function' => 'bb_nouveau_ajax_toggle_activity_notification_status',
+					'nopriv'   => true,
```

Comment:
@kishanbuddyboss This action is intended for users who are logged in.

Reply:
Fixed